.travis.yml

```diff
@@ -4,10 +4,14 @@ matrix:
   fast_finish: true
   include:
     - python: 3.6
-      env: TOXENV=py36
+      env:
+        - TOXENV=py36
+        - SKIP_NETWORK_TESTS=0
```

Comment:
I'm concerned about adding a network requirement for these tests. It means that they will fail if we can't reach archive (unlikely) or if autograph dev is down or broken (much more likely). What value are these adding that isn't covered by plain unit tests? Our typical way of doing integration testing is more spot-checky on branches such as maple.

These tests could still function as written without the network if the MARs and APks were put into the repo, and the autograph response was mocked -- I'm not sure if that adds any value on top of the existing unit tests, though.

I could see an argument for maybe running these in a completely separate job -- but that would still fail CI checks (and block merging) if things are down...so I'm not sure that's great either.

I guess I have a couple of specific concerns:
* What happens when we need to land some changes to this repo, and autograph is busted, which either slows down, prevents us from, or reduces confidence in something that we need to urgently do.
* The philosophical leap from standalone unit tests to tests-that-depend-on-the-network.

Reply:
I'm also not sure we're going to keep the dev instance around, since it was a one-off for testing.

Instead of keeping mocks up to date, I'd recommend running autograph with docker or docker compose like AMO does: https://github.com/mozilla/addons-server/blob/61d8bc0f40eed0b1706ed32221fff637742040a7/.travis.yml#L67-L69 (though use latest so we know if we break something)

Reply:
Great! I'll use the docker config instead!